### PR TITLE
UNF Embiggen

### PR DESCRIPTION
**Pull request by tehdiplomat**

### CODE REVIEW

File: forge-gui/res/cardsfolder/e/embiggen.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Embiggen
+ManaCost:G
+Types:Instant
+A:SP$ Pump | Cost$ G | ValidTgts$ Creature.nonBrushwagg | TgtPrompt$ Select target non Brushwagg creature | NumAtt$ +X | NumDef$ +X | SpellDescription$ Until end of turn, target non-Brushwagg creature gets +1/+1 for each supertype, card type, and subtype it has.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/e/embiggen.txt:4]: nice work
just picky `non Brushwagg` -> `non-Brushwagg`

**Comment by Agetian** [comment]
Good job! Can you pkease resolve the conflict so that I can merge it?

**Comment by Agetian** [comment]
Hmm, I tried rebasing on master, but that had a weird effect of adding a ton of commits to this branch :/
Can you please try on your side? Maybe a force push would be needed, not sure :/

**Comment by Agetian** [comment]
Will close this MR to avoid confusion, the merge is continued in #5156

**Pull request closed without merging**